main.py

```diff
@@ -60,11 +60,15 @@ def rename_status(id):
     return data_handler.rename_status(status_data)
 
 
-@app.route('/card/<id>', methods=['PATCH'])
+@app.route('/card/<id>', methods=['PATCH', 'DELETE'])
 @json_response
 def rename_card(id):
```

Comment:
ez a függvény azt az egy dolgot csinálja, amire a neve utal? :) (tudom, gonosz a kérdés, és nehéz is szépen elnevezni...)